templates/contact-us/newsletter-opt-in.html

```diff
@@ -0,0 +1,142 @@
+{% extends "contact-us/base_contact-us.html" %}
+
+{% block title %}Newsletter signup{% endblock %}
+
+{% block meta_copydoc %}
+{% endblock meta_copydoc %}
+
+{% block content %}
+  <div class="p-strip u-fixed-width">
+    <header style="border-bottom: 0; display: block;">
+      <h2 class="u-sv1" id="modal-title">Sign up to hear more about Ubuntu</h2>
+      <p>Get the latest news about Ubuntu and Canonical and make sure you never miss a new Ubuntu release.</p>
+    </header>
+    <form action="/marketo/submit"
+          method="post"
+          id="mktoForm_4960"
+          class="modal-form">
+      <ul class="p-list">
+        <li class="p-list__item">
+          <label for="email">Email address:</label>
```

Comment:
I actually chose to ignore the doc, because this doesn't seem like a situation you would want to put the work email. I will check with Rhys, you can follow the conversation here: https://docs.google.com/document/d/12BqHqrbgCwYEllFeR9yTd07cWojwz22Ao6dteHV98tI/edit?disco=AAABPdVRfpQ

Reply:
We settled on just 'Email'